src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -83,33 +83,30 @@ const IconOverImage = styled(Icon)`
   width: 50%;
 `;
 
-const StyledTabPanel = styled(TabPanel)`
-  margin-top: 1.5rem;
-`;
+const StyledTabPanel = styled(TabPanel)``;
 
-const ActionsTabs = styled(Tabs)`
-  margin-top: 1.2rem;
-`;
+const ActionsTabs = styled(Tabs)``;
```

Comment:
Do we actually need to have this ones as Styled if we are just going to use the base component?

Reply:
Just a renaming. Doesn't hurt tho, I would leave it for another pr to fix it in all the app

Reply:
MM disagree, we'll forget about this in a next pr

Reply:
What I mean is, there is a lot of these in the app, I would not adress a tech design in this pr, and not just for this specific, what I think we should do is decide if we don't like this approach and create a refactor for all the empty styled comps in the app. I don't like to modify our approach in just 1 place